showcase/src/app/news/news.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { browser } from "protractor";
```

Comment:
This e2e test can be removed for the moment since we haven't worked yet on the E2E testing.

We will create all these tests later ;)